packages/service/src/loopback-datasource-juggler.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/service
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+// tslint:disable:no-any
+
+export declare namespace juggler {
+  /**
+   * DataSource instance properties/operations
+   */
+  export class DataSource {
```

Comment:
Isn't this type definition shared with `@loopback/repository`? Can we move commonly-used type definitions directly into loopback-datasource-juggler please? 

My understanding is that we need to create `index.d.ts` file alongside existing [index.js](https://github.com/strongloop/loopback-datasource-juggler/blob/1ef38a6929ae100c73030b720097ba5c4b90a7b3/index.js) and place all type definitions there.

Reply:
That's on my plate to move TS type definitions into `loopback-datasource-juggler`. Let's deal it in a separate PR.